### ISSUE TITLE
Just poped up this message:

### ISSUE THREAD

**Issue by Manolouis** [cvzi/genius-lyrics-userscript#86]
![image](https://github.com/user-attachments/assets/15b043c0-4c5f-47ed-ac10-a91db5fbf852)

With this info in the info box:

themeName:  Spotify
Error:      Neither "Lyrics__Container" nor "LyricsPlaceholder__Container" found
URL:        https://open.spotify.com/collection/tracks
Genius:     https://genius.com/Beach-house-space-song-lyrics
Song:       Space Song by Beach House
Browser:    Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36

**Comment by cvzi** [comment]
Thanks, could you check if it still happens with this song? If yes, can you go in the Options of the script and click to clear the cache. And then reload (F5) the page. Maybe that helps. I forgot to clear the cache automatically with the last update.

**Comment by FurTactics** [comment]
Hi @cvzi
I think I have the same problem. Clearing cache didn't help. As far as I understood, Spotify changed some tag names in html.
Your script is very useful, especially with dark theme, and I'm really looking forward to your new version :sos:

**Comment by Manolouis** [comment]
nope, it works now for sm reason lol. Thanks for the response tho .🗿 goated developer.
